### PR TITLE
Update README.md for 1.4

### PR DESCRIPTION
**Pull request by hrydgard**
It's release time again! I'm aiming for next week.

Please add (or comment about) anything major that's missing from the What's New section - this is just an initial draft.

### CODE REVIEW

**Comment by DarkLegend481** [comment]
UWP? Yeah, I know... I am boring, every time talking about UWP... But, I can't wait to play PSP on my Lumia 635!!

**Comment by zminhquanz** [comment]
what about UWP on Windows 10 Mobile

**Comment by unknownbrackets** [comment]
I'm not sure if any of these are really worth mentioning, but...

 * Support for Maemo, Blackberry, and Symbian devices has been dropped. (#9055, #9050, #9051)
 * High DPI displays (like 4K) now render the UI properly. (#9216)
 * New setting to improve Bluetooth headset playback. (#9138)
 * Various improvements to desktop gamepad support. (#9268, #9346, #9124, #9330, #9260, #8993)

-[Unknown]

**Comment by ThePhoenixRises** [comment]
Animation of Pop up messages, to give you tips on some options.

**Comment by DarkLegend481** [comment]
Just updating my commentary:

• UWP Support for Windows 10 Mobile and PC (Closes https://github.com/hrydgard/ppsspp/issues/9164)

File: README.md [hunk 1]
```diff
@@ -25,8 +25,13 @@ What's new in 1.4.0
 * Hardware spline/bezier tesselation in OpenGL, D3D11 and Vulkan (...)
 * Post-processing shaders in D3D11
 * Prescale UV setting removed, now the default (improves perf) (#9176)
+* High DPI display fixes
+* New audio setting to improve compatibility with Bluetooth headsets
+* Various desktop gamepad compatibility fixes
 * Assorted minor compatibility fixes, code cleanup and performance improvements
 
+Support for Symbian, Maemo and Blackbarry has been removed.
```
Comment (unknownbrackets) [inline README.md]: BlackBARRy.

-[Unknown]

Comment (hrydgard) [inline README.md]: "Barry, is that how you get ants? Yes it is other Barry, yes it is."

I'll fix.

**Comment by xebra** [comment]
HW tess on Android causes a crash with some devices. So we should write down the notice about the issue, I think.
I implemented to check the extensions for unsupported device however GPU extension is supported but still some device not working correctly.

So I googled and found some information about instanced rendering issue on Android. It's that some mobile device GPU driver is something buggy when compile shaders with `gl_InstanceID`.

**Comment by zminhquanz** [comment]
@xebra i think i can hide this setting on ANDROID , if HW Tess completed and no issue on mobile , i will show this setting again

**Comment by xebra** [comment]
Anyway, it's not all devices, some modern devices are working correctly probably.
I'm not sure it should hide or not.
I think better solution is to hide using black/white list for device name(zenfone/galaxy/etc...).

**Comment by zminhquanz** [comment]
@xebra powervr have issue for this , isn't have `gl_instanced`

**Comment by xebra** [comment]
@zminhquanz  I think you have no problem about this. Your problem is just simply, your device is NOT supported probably.
I'm bothering issue that crashes with SUPPORTED device.

**Comment by xebra** [comment]
Probably, I got the solution to avoid crashes. I'll fix the checks of extensions soon.

**Comment by zminhquanz** [comment]
I think it's good for now , so ready to release 1.4 build ?

**Comment by hrydgard** [comment]
Soon!

**Comment by hrydgard** [comment]
Release will be tomorrow. Merging this now.

**Comment by hrydgard** [comment]
There's no UI for it yet, so adding that to the list will probably create questions :)

**Comment by sum2012** [comment]
Ok

**Comment by sum2012** [comment]
Improve UMD switch compatible, #9245 #9256

**Comment by hrydgard** [comment]
Thanks @sum2012 , added.

**Pull request merged**